src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/actions/ActionsSummary.java

```diff
@@ -0,0 +1,96 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.persistence.actions;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.persistence.ValueColumn;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Action summary that can be persisted in SQL
+ *
+ * <p>Table name : ActionsSummary
+ *
+ * <p>schema :
+ * | id(primary key) |     actionName       |  resourceValue    | nodeId    |
+ * |      1          | ModifyQueueCapacity  |      1            |  node1    |
+ * | nodeIp   | actionable | coolOffPeriod  |     timestamp
+ * | 127.0.0.1 |  true      |  300          |   1599257910923
+ */
+public class ActionsSummary {
+    private static final Logger LOG = LogManager.getLogger(ActionsSummary.class);
+
+    @ValueColumn public String actionName;
+    @ValueColumn public long resourceValue;
+    @ValueColumn public String id;
+    @ValueColumn public String ip;
+    @ValueColumn public boolean actionable;
+    @ValueColumn public long coolOffPeriod;
+    @ValueColumn public long timestamp;
+
+    public void setValues(
+            String actionName, long resourceValue, String id, String ip, boolean actionable, long coolOffPeriod, long timestamp) {
+        this.actionName = actionName;
+        this.resourceValue = resourceValue;
+        this.id = id;
+        this.ip = ip;
+        this.actionable = actionable;
+        this.coolOffPeriod = coolOffPeriod;
+        this.timestamp = timestamp;
+    }
+
```

Comment:
Do we really need this as we have setters/getters for individual fields ?

Reply:
Added it as it makes the code cleaner. Modified to use getters to set the values